docker/Dockerfile.linux.amd64

```diff
@@ -1,9 +1,12 @@
-FROM plugins/base:multiarch
+FROM alpine:3.12
 
 LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
   org.label-schema.name="Drone GCS" \
   org.label-schema.vendor="Drone.IO Community" \
   org.label-schema.schema-version="1.0"
 
+RUN mkdir -p /tmp
+RUN chmod 777 /tmp
```

Comment:
the alpine image already has a `/tmp` directory present.  I'm not sure what permissions, but this may not be necessary.

Reply:
Everyone gets rwx for /tmp directory in alpine. Removed it.